src/error.rs

```diff
@@ -73,6 +80,9 @@ impl IntoResponse for HcHttpGatewayError {
                     )),
                 )
             }
+            HcHttpGatewayError::UpstreamUnavailable => {
```

Comment:
Not handling `HolochainError` here, I think that's a job for the zome calls task